config/packages/easy_admin/conference.yaml

```diff
@@ -48,7 +63,22 @@ easy_admin:
                     - { property: 'cfpEndAt', type: 'date', 'type_options': { 'widget': 'single_text' } }
                     - { type: 'group', columns: 4, icon: 'cubes', label: 'Metadata' }
                     - { property: 'articleUrl', type: 'url' }
-        ExcludedConferences:
+            show:
+                fields:
+                    - { property: name, type: text }
+                    - { property: country, type: country }
+                    - { property: city, type: text }
+                    - { property: source, type: text }
+                    - { property: description, type: text}
+                    - { property: startAt, type: date }
+                    - { property: endAt, type: date }
+                    - { property: cfpUrl, type: url }
+                    - { property: cfpEndAt, type: date }
+                    - { property: siteUrl, type: url }
+                    - { property: articleUrl, type: url }
+                    - { property: submits, label: 'Submissions', type: embedded_list, css_class: d-block mx-auto }
```

Comment:
Could you add `"` around the `css_class` values ? It will avoid unexpected YAML wrong parsing